Runtime/Managers/Manager.cs

```diff
@@ -40,38 +42,85 @@ public abstract class Manager : MonoBehaviour
             return(s_Managers.ContainsKey(typeof(T)));
         }
 
-        static readonly Type[] kAllManagerTypes = TypeUtility.GetConcreteTypes<Manager>();
+        static readonly Type[] kAllManagerTypes = TypeUtility.GetConcreteTypes<Manager>();   
 
         [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.BeforeSceneLoad)]
         static void AutoCreateAll()
         {
             s_Managers.Clear();
 
-            var exclusionList = GameplayIngredientsSettings.currentSettings.excludedeManagers;
+            // Get the current scene Resources/ path
+            string sceneAssetPath = PathUtils.GetCurrentSceneResourcesPath();
 
-            if(GameplayIngredientsSettings.currentSettings.verboseCalls)
-                Debug.Log("Initializing all Managers...");
+            var inclusionList = GameplayIngredientsSettings.currentSettings.includedManagers;
+            var exclusionList = GameplayIngredientsSettings.currentSettings.excludedManagers;
```

Comment:
Please add a `[FormerlySerializedAs("excludedeManagers")]` attribute in order to prevent breaking user projects